ignite/contrib/metrics/regression/_base.py

```diff
@@ -47,20 +47,18 @@ def __init__(self, compute_fn, output_transform=lambda x: x):
         EpochMetric.__init__(self, compute_fn=compute_fn, output_transform=output_transform)
 
     def reset(self):
-        self._predictions = torch.tensor([], dtype=torch.float32)
-        self._targets = torch.tensor([], dtype=torch.float32)
+        self._predictions = []
```

Comment:
@hoangmit now we can remove `reset()` method as it is exactly the same as in `EpochMetric`.